eyeshade/lib/queries.js

```diff
@@ -37,3 +39,18 @@ function settlements (options = {}) {
  order by paid ${order}
  limit $2;`
 }
+
+function findTransactionByToken () {
+  return `
+ select * from transactions
+ where transaction_type = 'ad_payout' and to_account = $1::text and document_id = $2::text
+ limit 1;
+`
+}
+
+function insertTransaction () {
```

Comment:
I think the sql queries for inserting transactions are distinct enough to justify simply in-lining them vs abstracting into a separate queries library